backend/src/main/java/com/redhat/cloud/notifications/routers/models/behaviorgroup/CreateBehaviorGroupRequest.java

```diff
@@ -3,21 +3,36 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.databind.PropertyNamingStrategies;
 import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import org.apache.commons.lang3.StringUtils;
 
+import javax.validation.constraints.AssertFalse;
 import javax.validation.constraints.NotNull;
 import java.util.List;
 import java.util.Set;
 import java.util.UUID;
 
+/**
```

Comment:
This is a Javadoc kind of comment. It feels like a simple comment would be enough. WDYT?
```suggestion
/*
```

Reply:
I moved the comment from the request's class to the handler's method, since I think it makes more sense to have it there, close to the logic of how the creation is dealt with.